### ISSUE TITLE
Improve error message if no SDK version is selected

### ISSUE THREAD

**Issue by hbugdoll** [tidev/vscode-titanium#1020]
### Describe the Bug
If no Titanium SDK version is selected, no clear error message is shown in VSCode.

### Steps to reproduce the Bug
1. Have several SDK versions installed and one selected.
2. Delete the selected SDK version (and forget to run `titanium sdk select` to select another installed SDK version).
3. Try to build a project for a simulator or a device.
4. Neither simulators nor devices are listed in the Titanium extension tree. The less helpful error message _"Cannot read properties of undefined (reasing 'simulators')"_ occurred after refreshing devices.
5. Running `titanium` on the terminal shows the dilemma:
```
[ERROR] Invalid selected Titanium SDK ...
[ERROR] Run "titanium sdk select" to set the selected Titanium SDK
```

### Expected Behavior
Show directly the error message _"Invalid selected Titanium SDK ..."_ in VSCode.

### Your Environment
| Software | Version |
| ---------------- | ---------- |
| Titanium SDK | 10.1.1.GA / 11.0.0.GA |
| VS Code | 1.69.2 |
| vscode-titanium | 1.0.0 |

**Comment by caspahouzer** [comment]
Please update to the latest CLI with

npm i -g titanium

and try if the error comes up again.

**Comment by hbugdoll** [comment]
@ewanharris That's great news. Thanks.

**Comment by ewanharris** [comment]
This was implemented in #1028 and shipped in v1.1.0
